AutoRest/Generators/CSharp/Azure.CSharp/Properties/AssemblyInfo.cs

```diff
@@ -4,7 +4,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
+// General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("AutoRest")]
```

Comment:
also we can fill in more for several other fields below, such as "AssemblyCompany"